docs/adrs/0002-field-for-data-type-form-information.md

```diff
@@ -0,0 +1,162 @@
+# 1. Field for Data Type / Form Information
+
+Follows: https://mitlibraries.github.io/guides/misc/adr.html
+
+Date: 2024-02-15
+
+## Status
+
+Proposed
+
+## Context
+
+### Need
+
+Identify a pre-existing, or new, TIMDEX field to store information about GIS datasets that will support required facet filters, while also potentially serving other TIMDEX data sources.
+
+### Overview
+During a project that introduced geospatial data into TIMDEX, two new sources were added `gisogm` and `gismit`, both of which use the new Transmogrifier transformer 
+`MITAardvark`. Records for these sources come from the [GeoHarvester](https://github.com/MITLibraries/geo-harvester) which produces [Aardvark metadata schema](https://opengeometadata.org/ogm-aardvark/) JSON records.    
+
+During work on a new geospatial TIMDEX UI, the question was raised as to what fields should be used for two new required facet filters to mirror functionality in the [current Geoweb application](https://geodata.mit.edu/):
+
+  * "Data Type"
+    * values like: `[Polygon, Point, Raster, Image, etc.]`
+  * "Format"
+    * values like: `[Shapefile, TIFF, GeoTIFF, JPEG, etc.]`
```

Comment:
Hmmm. I think that might be another option to document?

i.e. move `[Shapefile, TIFF, GeoTIFF, JPEG, etc.]` to file_formats... and `[Polygon, Point, Raster, Image, etc.]` slide into `format`... no data model changes required?

(I'm not saying I'd pick that option, but it feels like an option to document)

Reply:
I can write that up and push a commit, if that works

Reply:
Thanks for suggesting @ehanson8.  `file_formats` felt like a non-starter because other Transmog sources were writing mimetypes... but would probably be relatively easy to update them.

I'm liking that proposal quite a bit, but did want to note that would mean the "Format" filter in the UI would actually be coming from `file_formats` and "Data Type" filter would be coming from `format`.   Would probably want to document that in the UI why this mapping was selected.

For any other sources that write to `format`, I'm assuming we might want to consider if the values they are writing might then belong in `file_formats`?

Reply:
Pushed commit with the 4th option